### ISSUE TITLE
IntelliJ not inferring "new" macro from derive-new crate causes false positive "unresolved reference" error

### ISSUE THREAD

**Issue by saskenuba** [intellij-rust/intellij-rust#4959]
## Environment

* **IntelliJ Rust plugin version:** 0.2.117.2852-193-nightly
* **Rust toolchain version:** 1.43.0-nightly (58b834344 2020-02-05) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2019.3.4 (CL-193.6494.38)
* **Operating system:** Linux 5.5.2-arch1-1

Experimental macro engine is being used.

## Problem description

Rust plugin can't infer `fn new` after deriving `new` macro from `derive-new` crate.
It doesn't show up for completion and appears in red as unresolved reference.

Code compiles correctly.

## Steps to reproduce

Create a new Rust project, and replace the main/lib file with the following code:

```rust
#[macro_use]
extern crate derive_new;

#[derive(new)]
struct Test {
    #[new(value = "32")]
    number: i32
}

fn test() {
    Test::ne/*caret*/
    Test::new(); // in red
}
```

**Comment by Undin** [comment]
@saskenuba Unfortunately, the plugin doesn't support procedural macros at this moment

**Comment by Undin** [comment]
Duplicate of #1786
